datahub-web-react/src/app/shared/tags/TagTermGroup.tsx

```diff
@@ -81,6 +95,38 @@ export default function TagTermGroup({
             {domain && (
                 <DomainLink domain={domain} name={entityRegistry.getDisplayName(EntityType.Domain, domain) || ''} />
             )}
+            {uneditableAccess?.roles?.map((role) => {
+                console.log(role);
+                renderedTags += 1;
+                if (maxShow && renderedTags === maxShow + 1)
+                    return (
+                        <TagText>
+                            <Highlight matchStyle={highlightMatchStyle} search={highlightText}>
+                                {uneditableAccess?.roles ? `+${uneditableAccess?.roles?.length - maxShow}` : null}
+                            </Highlight>
+                        </TagText>
+                    );
+                if (maxShow && renderedTags > maxShow) return null;
+                return (
+                    <div>
+                        {console.log(role?.role)}
```

Comment:
please remove...

Reply:
removed the changes as we cannot make changes in the default preview card